### PR TITLE
Integration of Thunder ProcessContainer Support

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: dynamic-layers/virtualization/recipes-kernel/linux/linux-yocto_virtualization.inc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/linux-yocto:"
+
+SRC_URI += "file://xt-checksum.scc \
```
Comment (artur-gebicz-ml) [inline dynamic-layers/virtualization/recipes-kernel/linux/linux-yocto_virtualization.inc:3]: Could you please align the lines?

File: recipes-containers/dobby/dobby.bb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+#2020-12-14
+SRCREV = "2db374a248e1e011a13d7ba643901f7da6972710"
+DEPENDS_append = " libnl dbus jsoncpp python3 ctemplate boost yajl systemd"
+RDEPENDS_${PN} = "crun (>= 0.14.1)"
+
+S = "${WORKDIR}/git"
+
+# Always build the debug version for now
+EXTRA_OECMAKE = "-DCMAKE_BUILD_TYPE=Debug "
```
Comment (artur-gebicz-ml) [inline recipes-containers/dobby/dobby.bb:16]: Could you please make it a PACKAGECONFIG?

File: recipes-containers/dobby/dobby.bb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+# Sky XiOne build
+EXTRA_OECMAKE_append_skyxione = "-DLEGACY_COMPONENTS=ON -DRDK_PLATFORM=XI1 "
```
Comment (artur-gebicz-ml) [inline recipes-containers/dobby/dobby.bb:19]: Could you please make it a PACKAGECONFIG?

File: recipes-containers/dobby/dobby.bb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+# Sky Llama build
+EXTRA_OECMAKE_append_llama = "-DLEGACY_COMPONENTS=ON -DRDK_PLATFORM=LLAMA "
```
Comment (artur-gebicz-ml) [inline recipes-containers/dobby/dobby.bb:22]: Could you please make it a PACKAGECONFIG?

File: recipes-containers/dobby/dobby.bb [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+# Comcast Xi6 build
+EXTRA_OECMAKE_append_arrisxi6 = "-DLEGACY_COMPONENTS=OFF -DRDK_PLATFORM=XI6 "
```
Comment (artur-gebicz-ml) [inline recipes-containers/dobby/dobby.bb:25]: Could you please make it a PACKAGECONFIG?

File: recipes-core/ctemplate/ctemplate_git.bb [hunk 6]
```diff
@@ -0,0 +1,26 @@
+SUMMARY = "ctemplate"
```
Comment (artur-gebicz-ml) [inline recipes-core/ctemplate/ctemplate_git.bb:1]: Is it native only package? If yes, please add -native to the name of the file (like ctemplate-native_git.bb)

File: recipes-wpe/wpeframework/wpeframework_git.bb [hunk 7]
```diff
@@ -19,9 +19,6 @@ RPROVIDES_${PN} += "thunder"
 
 WPEFRAMEWORK_SYSTEM_PREFIX ??= "WPE"
 
-# Configure exceptions handling (ON/OFF)
-WPEFRAMEWORK_EXCEPTIONS_ENABLE ??= "OFF"
```
Comment (artur-gebicz-ml) [inline recipes-wpe/wpeframework/wpeframework_git.bb:23]: What's the reason for deleting this flag?

File: recipes-wpe/wpeframework/wpeframework_git.bb [hunk 8]
```diff
@@ -107,12 +113,10 @@ EXTRA_OECMAKE += "\
     -DEXTERN_EVENTS="${WPEFRAMEWORK_EXTERN_EVENTS}" \
     -DBUILD_SHARED_LIBS=ON \
     -DRPC=ON \
-    -DVIRTUALINPUT=ON  \
```
Comment (artur-gebicz-ml) [inline recipes-wpe/wpeframework/wpeframework_git.bb:110]: Could you please tell me why this was removed?

**Review by artur-gebicz-ml — CHANGES_REQUESTED** [review]

File: recipes-wpe/wpeframework/wpeframework_git.bb (hunk 7 above)
Comment (HaseenaSainul) [reply recipes-wpe/wpeframework/wpeframework_git.bb:23]: It is already available as part of PACKAGECONFIG, hence removed it

File: recipes-wpe/wpeframework/wpeframework_git.bb (hunk 8 above)
Comment (HaseenaSainul) [reply recipes-wpe/wpeframework/wpeframework_git.bb:110]: It is already available as part of PACKAGECONFIG, hence removed it

File: recipes-core/ctemplate/ctemplate_git.bb (hunk 6 above)
Comment (HaseenaSainul) [reply recipes-core/ctemplate/ctemplate_git.bb:1]: It is required for target also

File: dynamic-layers/virtualization/recipes-kernel/linux/linux-yocto_virtualization.inc (hunk 1 above)
Comment (HaseenaSainul) [reply dynamic-layers/virtualization/recipes-kernel/linux/linux-yocto_virtualization.inc]: done

File: recipes-containers/dobby/dobby.bb (hunk 2 above)
Comment (HaseenaSainul) [reply recipes-containers/dobby/dobby.bb]: done

File: recipes-containers/dobby/dobby.bb (hunk 3 above)
Comment (HaseenaSainul) [reply recipes-containers/dobby/dobby.bb]: done

File: recipes-containers/dobby/dobby.bb (hunk 4 above)
Comment (HaseenaSainul) [reply recipes-containers/dobby/dobby.bb]: done

File: recipes-containers/dobby/dobby.bb (hunk 5 above)
Comment (HaseenaSainul) [reply recipes-containers/dobby/dobby.bb]: done

**Review by artur-gebicz-ml — APPROVED** [review]

**Pull request merged**